src/components/App.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import ArtistContainer from './ArtistContainer'
+const App = () => {
+    return <div><ArtistContainer /></div>
+  };
+
+export default App
```

Comment:
@aonomike The `Artist` container won't be placed directly into the `App` component.
 
Instead, use routing within `App` (`react-router-dom`), to provide a route for it.
See Stephen Grider's lectures 194 and forward or the following lecture on Links in Routes:
https://www.udemy.com/react-redux/learn/v4/t/lecture/12700571?start=0
 
You'll want to use something like this for the Artist:
`<Route path="/artists/:id" component={ArtistContainer} />`

Reply:
Sure! So the nav stays outside in the app right where we route the rest of the links?

Reply:
@javpet Yep. It can be a sibling component to the `<Route />` components.